libraries/azure_security_center_policy.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'azurerm_resource'
+
+class AzureSecurityCenterPolicy < AzurermResource
+  name 'azure_security_center_policy'
+  desc 'Verifies settings for Security Center'
+  example <<-EXAMPLE
```

Comment:
I notice we use both `<<-EXAMPLE` and `"` should we standardize?

Reply:
Generally we've used the HEREDOC syntax (`<<-EXAMPLE`). I think we should standardize on that instead of the multi-line string.

Reply:
I fixed so we use HEREDOCs everywhere.

Reply:
You da best.